jobs/Backend/Task/src/Mews.ExchangeRateProvider/Mappers/CzechNationalBankExchangeRateMapper.cs

```diff
@@ -0,0 +1,25 @@
+using FileHelpers;
+
+namespace Mews.ExchangeRateProvider.Mappers;
+
+/// <summary>
+/// Maps pipe symbol delimited raw text data from the exchange rate website onto an ExchangeRate model
+/// </summary>
+public sealed class CzechNationalBankExchangeRateMapper
+{
+    private const string CzechKorunaCurrencyCode = "CZK";
```

Comment:
I've made an assumption here that all exchange rate requests will wish to be compared to the base rate of CZK - if this isn't appropriate then some re-working will be required.